pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/DemoApplication.kt

```diff
@@ -2,29 +2,32 @@
  * Copyright (c) 2023. SRG SSR. All rights reserved.
  * License information is available from the LICENSE file.
  */
-package ch.srgssr.pillarbox.demo.di
+package ch.srgssr.pillarbox.demo
 
-import android.content.Context
+import android.app.Application
 import ch.srgssr.pillarbox.analytics.AnalyticsConfig
 import ch.srgssr.pillarbox.analytics.SRGAnalytics
 import ch.srgssr.pillarbox.analytics.commandersact.CommandersAct
 
 /**
- * Analytics module
+ * Demo application
+ *  - Init SRG Analytics
  */
-object AnalyticsModule {
-    private const val VIRTUAL_SITE = "pillarbox-demo-android"
+class DemoApplication : Application() {
 
-    /**
-     * Provider analytics
-     */
-    fun providerAnalytics(appContext: Context): SRGAnalytics {
+    override fun onCreate() {
+        super.onCreate()
+        initAnalytics()
+    }
+
+    private fun initAnalytics() {
         val analyticsConfig = AnalyticsConfig(
             distributor = AnalyticsConfig.BuDistributor.SRG,
             nonLocalizedApplicationName = "PillarboxDemo"
         )
-        val commandersActConfig = CommandersAct.Config(virtualSite = VIRTUAL_SITE, sourceKey = CommandersAct.Config.SOURCE_KEY_SRG_DEBUG)
+        val commandersActConfig = CommandersAct.Config(virtualSite = "pillarbox-demo-android", sourceKey = CommandersAct.Config.SOURCE_KEY_SRG_DEBUG)
```

Comment:
Would the `virtualSite` and the `sourceKey` shouldn't be on the `Analytics.Config`?